src/WebJobs.Script.WebHost/Middleware/HostWarmupMiddleware.cs

```diff
@@ -57,7 +58,7 @@ private void PreJitPrepare(string jitTraceFileName)
 
             if (file.Exists)
             {
-                JitTraceRuntime.Prepare(file, out int successfulPrepares, out int failedPrepares);
+                JitTraceRuntime.Prepare(file, out List<string> successfulPrepares, out List<string> failedPrepares);
```

Comment:
I'm in the process of updating the jittrace file for these changes. I made some changes to this API to make it easier to see the full list of failed prepares.